homeassistant/components/plugwise/switch.py

```diff
@@ -78,7 +103,7 @@ def __init__(
     @property
     def is_on(self) -> bool | None:
```

Comment:
I just spotted this (also applies to `binary_sensor`).
If the description always returns a `bool`, then `is_on` should also always return a `bool`.
```suggestion
    def is_on(self) -> bool:
```

Reply:
Let me double-check that with @bouwew

Reply:
Yes, the None option can be removed.
Both for `binary_sensor` and `switch`.

Reply:
I'll create a follow-up PR for binary_sensor to keep in line with the small PR strategy